api/src/main/java/marquez/cli/DbRetentionCommand.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2018-2023 contributors to the Marquez project
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package marquez.cli;
+
+import io.dropwizard.cli.ConfiguredCommand;
+import io.dropwizard.db.DataSourceFactory;
+import io.dropwizard.db.ManagedDataSource;
+import io.dropwizard.setup.Bootstrap;
+import lombok.NonNull;
+import lombok.extern.slf4j.Slf4j;
+import marquez.MarquezConfig;
+import marquez.db.DbRetention;
+import marquez.db.exceptions.DbRetentionException;
+import net.sourceforge.argparse4j.inf.Namespace;
+import org.jdbi.v3.core.Jdbi;
+import org.jdbi.v3.postgres.PostgresPlugin;
+
+/**
+ * A command to apply retention policy to source, dataset, and job metadata collected by Marquez.
+ *
+ * <h2>Usage</h2>
+ *
+ * For example, to override the {@code retention-days}:
+ *
+ * <pre>{@code
+ * java -jar marquez-api.jar db-retention --retention-days 14
+ * }</pre>
+ */
+@Slf4j
+public class DbRetentionCommand extends ConfiguredCommand<MarquezConfig> {
+  private static final String DB_SOURCE_NAME = "db-retention-source";
+
+  /* Default retain for metadata. */
+  private static final int DEFAULT_RETENTION_DAYS = 7;
+
+  /* Args for db-retention command. */
+  private static final String CMD_ARG_RETENTION_DAYS = "retentionDays";
+
+  /* Define db-retention command. */
+  public DbRetentionCommand() {
+    super("db-retention", "apply retention policy to database");
+  }
+
+  @Override
+  public void configure(@NonNull final net.sourceforge.argparse4j.inf.Subparser subparser) {
+    super.configure(subparser);
+    subparser
+        .addArgument("--retention-days")
+        .dest(CMD_ARG_RETENTION_DAYS)
+        .type(Integer.class)
+        .required(false)
+        .setDefault(DEFAULT_RETENTION_DAYS)
+        .help("the number of days to retain metadata");
```

Comment:
Would it make sense to add, as part of the help, that this is a one-off run of the command (the user may assume this changes the config for them, not sure - people can be very creative :))

Reply:
I can see how the cmd description can be misleading and allow for creative interpreation. I've updated the cmd usage to include 

```
usage: java -jar marquez-api.jar
       db-retention [--chunk-size CHUNKSIZE] [--retention-days RETENTIONDAYS] [-h] [file]

apply one-off ad-hoc retention policy directly to database

...
```